### PR TITLE
#318 Add ESLint support and configure it to use Drupal standards.

### PR DESCRIPTION
**Pull request by hkirsman**
*How to test:*

1. Get this branch
    `git clone git@github.com:wunderio/drupal-project.git`
    `cd drupal-project`
    `git checkout feature/#318-Add-ESLint-support-and-configure-it-to-use-Drupal-standards`
2. Start Lando
   ` lando start` (or `lando rebuild` if you've working on existing Lando project)
3. Create example js file
    `echo 'var foo="bar"' > web/modules/custom/phpunit_example/example.js`
4. Add the file to GIT
    `git add web/modules/custom/phpunit_example/example.js`
5. Try to commit
    `git commit`
6. Allow it to fix the script automatically and check the results

There's also an alternative PR here https://github.com/wunderio/drupal-project/pull/320 that uses node service but it's currently proof of work. It breaks Code Quality scans, you need to configure name and e-mail in the node service.

### CODE REVIEW

**Review by misterjoonas — APPROVED** [review]
Tested and works 👍 💯

File: .lando.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@ services:
     # User random static high port for database connection.
     portforward: 34567
   appserver:
+    build_as_root:
+      # To be able to run ESLint with GrumPHP, we need Node.js on appserver.
+      - "apt-get update"
+      - "apt-get install -y nodejs"
```
Comment (tormi) [inline .lando.yml:45]: I would avoid such things in a template - they slow down the build time. Could we add `nodejs` to the Silta PHP image instead?

File: package-lock.json [hunk 2]
```diff
@@ -1,3 +1,2761 @@
 {
-  "lockfileVersion": 2
+  "name": "wunderio-drupal-project",
```
Comment (tormi) [inline package-lock.json:2]: So far we've avoided adding lockfile dependencies to the repo for maintenance reasons. See https://github.com/wunderio/drupal-project/issues/143 & https://github.com/wunderio/drupal-project/pull/155. Should we change this?

File: grumphp.yml [hunk 3]
```diff
@@ -23,6 +23,9 @@ grumphp:
     # PHPUnit will fail with 0 tests.
     phpunit:
       testsuite: unit
+    eslint:
```
Comment (tormi) [inline grumphp.yml:26]: Let's consider other options as well like using [Husky](https://typicode.github.io/husky/#/) via `node` service.

**Pull request closed without merging**